corehq/apps/userreports/templates/userreports/builder_report_type_select.html

```diff
@@ -37,38 +42,37 @@
         {% include 'app_manager/partials/new_app_buttons.html' %}
     {% else %}
         <div class="row-fluid">
-            <div class="span2">
-                <a href="{% url "report_builder_select_source" domain "chart" %}">
-                    <button class="btn btn-primary btn-large btn-report-type">
-                        {% trans 'Chart' %}
-                        <span class="hq-help-template" data-content="{% trans "A bar graph or a pie chart to show data from your cases or forms. You choose the property to graph." %}"></span>
-                    </button>
-                </a>
-            </div>
-            <div class="span2">
-                <a href="{% url "report_builder_select_source" domain "list" %}">
-                    <button class="btn btn-primary btn-large btn-report-type">
-                        {% trans 'Form or Case List' %}
-                        <span class="hq-help-template" data-content="{% trans "A list of cases or form submissions. You choose which properties will be columns."%}"></span>
-                    </button>
-                </a>
-            </div>
-            <div class="span2">
-                <a href="{% url "report_builder_select_source" domain "worker" %}">
-                    <button class="btn btn-primary btn-large btn-report-type">
-                        {% trans 'Worker Report' %}
-                        <span class="hq-help-template" data-content="{% trans "A table of your mobile workers. You choose which properties will be the columns." %}"></span>
-                    </button>
-                </a>
-            </div>
-            <div class="span2">
-                <a href="{% url "report_builder_select_source" domain "table" %}">
-                    <button class="btn btn-primary btn-large btn-report-type">
-                        {% trans 'Data Table' %}
-                        <span class="hq-help-template" data-content="{% trans "A table of aggregated data from form submissions or case properties. You choose the columns and rows." %}"></span>
-                    </button>
-                </a>
-            </div>
+            {% for tile in tiles %}
+            <tile-{{ tile.ng_directive }}
+                    slug="{{ tile.slug }}"
+                    title="{{ tile.title }}">
+                <div class="col-md-3">
```

Comment:
when I get rid of ```col-md-3``` it turns into

<img width="1394" alt="screen shot 2015-07-09 at 6 09 25 pm" src="https://cloud.githubusercontent.com/assets/1757035/8608092/bd244f84-2665-11e5-9813-0eed27b4790b.png">